browser/ui/tabs/brave_tab_prefs.cc

```diff
@@ -24,6 +25,16 @@ void RegisterBraveProfilePrefs(PrefRegistrySimple* registry) {
   registry->RegisterBooleanPref(kVerticalTabsFloatingEnabled, true);
   registry->RegisterIntegerPref(kVerticalTabsExpandedWidth, 220);
   registry->RegisterBooleanPref(kVerticalTabsOnRight, false);
+  registry->RegisterBooleanPref(kVerticalTabsShowScrollbar, false);
+}
+
+void MigrateBraveProfilePrefs(PrefService* prefs) {
+  if (auto* pref = prefs->FindPreference(kVerticalTabsShowScrollbar);
+      pref && pref->IsDefaultValue() &&
+      base::FeatureList::IsEnabled(
+          tabs::features::kBraveVerticalTabScrollBar)) {
+    prefs->SetBoolean(kVerticalTabsShowScrollbar, true);
```

Comment:
In the issue, it says we should default to being off rather than on. Apart from that this seems fine :smile:

Reply:
Yeah, that's right. This line would affect only if a user has turned on this feature from the flag. Otherwise, the default value is still false 😄